ultravox/tools/ds_tool/ds_tool.py

```diff
@@ -128,7 +128,8 @@ def map_split(
 
         # Filter out samples where new_column_name is None
         return ds_mapped.filter(
-            lambda sample: sample[self.new_column_name] != "",
+            lambda sample: sample[self.new_column_name] is not None
+            and sample["audio"] is not None,
```

Comment:
Why the `sample["audio"] is not None`?
Specially since this is part of the `TextGenerationTask`.

Reply:
I suspect training wont work with datasets that have empty audio values

Reply:
`ds_tool` was meant to be used for different tasks and even multiple times. You can first generate some stuff, then add audio for those fields or others. Even if you do have an audio, it might be in a different column.

Reply:
Hm, then I suspect we might need to add another method that does some cleaning of the data. I can do that in a separate pr.